# **Objetivo**
Conocer la estructura inicial del dataset.

# **Código**

In [6]:
import pandas as pd
df = pd.read_csv( "FAMILIAS_FOCALIZADAS.csv" )
print(df.shape)
df.head()

(5543, 22)


,Bancarizado,CodigoDepartamentoAtencion,CodigoMunicipioAtencion,Discapacidad,EstadoBeneficiario,Etnia,FechaInscripcionBeneficiario,Genero,NivelEscolaridad,NombreDepartamentoAtencion,...,TipoAsignacionBeneficio,TipoBeneficio,TipoDocumento,TipoPoblacion,RangoBeneficioConsolidadoAsignado,RangoUltimoBeneficioAsignado,FechaUltimoBeneficioAsignado,RangoEdad,Titular,CantidadDeBeneficiarios
0,NO,15,15572,SI,ACTIVO,ND,2012-12-01,Mujer,ND,BOYACA,...,ND,ND,TI,DESPLAZADOS,0 - 1.500.000,0 - 1.300.000,1900-01-01,06-17,NO,1
1,NO,15,15572,NO,ACTIVO,ND,2013-08-01,Mujer,PRIMARIA,BOYACA,...,ND,EDUCACIÓN PRIMARIA,TI,DESPLAZADOS,0 - 1.500.000,0 - 1.300.000,1900-01-01,06-17,NO,3
2,ND,15,15572,NO,ACTIVO,ND,2015-09-01,Mujer,ND,BOYACA,...,MONETARIO,NUTRICIÓN,No Definido,DESPLAZADOS,3.000.001 - 4.500.000,0 - 1.300.000,2018-01-01,30-49,SI,1
3,SI,15,15572,NO,ACTIVO,ND,2012-12-01,Mujer,ND,BOYACA,...,MONETARIO,ND,CC,UNIDOS,0 - 1.500.000,0 - 1.300.000,2018-01-01,>65,SI,1
4,SI,15,15572,NO,ACTIVO,ND,2012-12-01,Mujer,ND,BOYACA,...,MONETARIO,ND,CC,UNIDOS,0 - 1.500.000,0 - 1.300.000,2018-01-01,>65,SI,1


# **Preguntas**

*   ¿Cuántas columnas tiene el dataset?
    El dataset tiene **22 columnas**.
*   ¿Qué variables contiene?
    El dataset contiene las siguientes variables:
>    `Bancarizado`, `CodigoDepartamentoAtencion`, `CodigoMunicipioAtencion`, `Discapacidad`, `EstadoBeneficiario`, `Etnia`, `FechaInscripcionBeneficiario`, `Genero`, `NivelEscolaridad`, `NombreDepartamentoAtencion`, `TipoAsignacionBeneficio`, `TipoBeneficio`, `TipoDocumento`, `TipoPoblacion`, `RangoBeneficioConsolidadoAsignado`, `RangoUltimoBeneficioAsignado`, `FechaUltimoBeneficioAsignado`, `RangoEdad`, `Titular`, `CantidadDeBeneficiarios`.
    
*   ¿Cuántos registros existen?
    El dataset contiene 5543 registros.




